### PR TITLE
Add YouTube Statistics and Add channel title

### PR DESCRIPTION
**Pull request by astrit**
## Description

Get your channel statistics like: views, subscribers, videos and channel title on Raycast dashboard

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

yt

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/dashboard/youtube-statistics.template.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# @raycast.title YouTube Statistics
+# @raycast.mode inline
+# @raycast.refreshTime 1h
+
+# Optional parameters:
+# @raycast.packageName YouTube Statistics
+# @raycast.icon images/youtube.png
+
+# Documentation:
+# @raycast.description Shows YouTube Subscribers
```
Comment (unnamedd) [inline commands/dashboard/youtube-statistics.template.rb:14]:
```suggestion
# @raycast.description Shows YouTube Subscribers, Views and Videos
```

File: commands/dashboard/youtube-statistics.template.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  use_ssl: uri.scheme == "https",
+}
+
+res = Net::HTTP.start(uri.hostname, uri.port, req_options) { |http|
+  http.request(req)
+}
+
+if res.code == "200"
+  result = JSON.parse(res.body)
+
```
Comment (unnamedd) [inline commands/dashboard/youtube-statistics.template.rb:44]: What about if you do something like

```suggestion
   item = result["items"][0] 
   statistics = item["statistics"]

   title = item["snippet"]["title"]
   subs = statistics["subscriberCount"]
   views = statistics["viewCount"]
   videos = statistics["videoCount"]
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @astrit,

just minor comments here, nothing critical, but would improve a bit the Script Command!

**Comment by astrit** [comment]
@unnamedd good idea actually I did update and now nothing else is open. Thank you very much for the suggestions. ✌️

**Comment by dehesa** [comment]
Thanks for the contribution @astrit. Could you please squash all your commits?

**Comment by astrit** [comment]
Hey @dehesa I did squash them please let me know if this is good ✌️

**Pull request closed without merging**